src/uncrustify.cpp

```diff
@@ -479,6 +480,36 @@ int main(int argc, char *argv[])
       }
    }
 
+   /* Set config options using command line arguments.*/
+   idx = 0;
+   while ((p_arg = arg.Params("--set", idx)) != NULL)
+   {
+      char buffer[256];
+      strcpy(buffer, p_arg);
+
+      char *ptr;
+      if ((ptr = strchr(buffer, '=')) != NULL)
+      {
+         // Tokenize the string using the null character and move to second token.
+         *ptr++ = '\0';
```

Comment:
This one seems to be also added in C11. On the other hand, the entire "algorithm" is a simplified version of `strtok`. I think it can be replaced with it.